docs/user-guide/systemrequirements-cli.md

```diff
@@ -1,49 +1,91 @@
 # Zowe CLI System requirements
 
-Before installing Zowe CLI, ensure that your environment meets the prerequisites that are described in this article.
+Before installing Zowe CLI, ensure that your environment meets the prerequisites that are described here.
+
 ## Client-side requirements
 
 Zowe CLI is supported on Windows, Linux, and Mac operating systems. Meet the following requirements before you install the CLI:
 
-- **Node.js:** Install a currently supported version of [Node.js LTS](https://nodejs.org/en/). For a complete list of supported LTS versions, see [Nodejs Releases](https://nodejs.org/en/about/releases/).
+### Node.js
+
+The JavaScript runtime environment Node.js must be installed to run JavaScript applications (such as Zowe CLI) outside of a web browser.
+
+To install Node.js:
+
+  1. Go to [Node.js LTS](https://nodejs.org/en/) to select and install a runtime version with active support.
 
-  **Note:** You might need to restart the command prompt after installing Node.js. Issue the following command to verify that Node.js is installed.
+  For a list of supported LTS versions, see [Nodejs Releases](https://https://nodejs.org/en/about/previous-releases).
+  2. Restart the command prompt after installing Node.js, if required.
+  3. Verify that Node.js is installed:
 
   ```
   node --version
   ```
+  Node.js is installed on your PC when Zowe CLI returns a message with the correct Node.js version.
+  
+  If you issue the `node --version` command and get an error message, confirm that your PATH environment variable includes the path to the Node.js installation location.
```

Comment:
@adam-wolfe -- can we provide a location for the PATH env variable?

Reply:
It's operating system-dependent. On Linux systems, you can enter `echo $PATH` into a terminal to print out the value. On Windows, you can edit environment variables in the System Properties settings window. Not sure about how to do it on Mac. It might be simpler to let the user figure this out for their own system.